### ISSUE TITLE
Stronghold Definitive Edition

### ISSUE THREAD

**Issue by NuclearPhoenixx** [sourcehold/Sourcehold#109]
The title pretty much says it all, Firefly Studios recently announced the release of the [Stronghold Definitive Edition](https://store.steampowered.com/app/2140020/Stronghold_Definitive_Edition/), which will (or at least is trying to be) be a remaster of Stronghold 1.

What do you think of that regarding this project specifically? Will you continue or is it already pretty much dead (looks like it)? The remaster will probably address all the points you wanted to improve upon on the original game. Well, maybe all except the open-source nature of course. And oh well, I hope they make a working game and don't just release another trashy game.

**Comment by gynt** [comment]
Yea good point! This project has achieved a lot: parsing assets, framework for a new implementation.
This project is on hold until we see what the Definitive Edition can do!

**Comment by drozumek** [comment]
Pretty dissapointed by Definitive Edition so far - same bugs, same poor AI, same gameplay annoying issues.

**Comment by NuclearPhoenixx** [comment]
Oh really? That's pretty sad, but it's got good reviews actually?

**Comment by DvP17** [comment]
I'm disappointed by the Definitive Edition, too. Firefly has just recycled their copyright. The original game actually looks better.

**Comment by NuclearPhoenixx** [comment]
> The original game actually looks better.

For real? 🤦🏻

**Comment by drozumek** [comment]
> > The original game actually looks better.
> 
> For real? 🤦🏻

No, Definitive Edition looks better but you barely can take advantage of new graphics - game has stupid zoom steps which one its too close, the other one is too fat so the game looks like stronghold HD. The is no zoom in between. 

Moreover - I am on mission 15 - AI is stupid as it was before, game is buggy. I returned the game - like someone above wrote - its just recycled Stronghold HD.

**Comment by NuclearPhoenixx** [comment]
Oh well, it's funny to see the different kinds of reactions. It definitely looks better and the zoom is pretty stupid in how there is not reason it's not stepless. However, I don't know what most of you were expecting. They took the old game, updated some aspects and left the other stuff alone. If they changed too much, there would be a shit storm of people complaining it's not the original game anymore. And considering the current state of the gaming industry and what crap (sorry) Firefly has produced in the recent years, I would definitely call that a win.

I also completely agree that they could have at least fixed all the old bugs, that's just sad and the game could have definitely benefited from a better AI. But to call it just a recycle is a stretch IMO. It's a re-release of the game with some added features, updated graphics and small changes, that's exactly what a definitive edition is supposed to be. Maybe you were expecting it to be a complete remake? That would've been cool, but yeah here we are...

**Comment by NuclearPhoenixx** [comment]
Sure, I definitely agree on that with you.

**Comment by gynt** [comment]
So what is the consensus? To me it seems like they didn't fix any internal bugs, only UI related bugs

**Comment by idchlife** [comment]
Stronghold Definitive Edition does not offer much in terms of graphics (except obvious hi res, more frames etc), but it introduces low FPS, high CPU / GPU temperatures and occasional stuttering for some reason.

I'm pretty disappointed because I have games in my library that look much better and runs much faster without the need for additional CPU/GPU cooling.

**Comment by NuclearPhoenixx** [comment]
> but it introduces low FPS, high CPU / GPU temperatures and occasional stuttering for some reason.

I have noticed that too, IMO the Crusader DE Demo runs much better than the Stronghold 1 DE, weird.
